### PR TITLE
add array_append process

### PR DESCRIPTION
**Pull request by zcernigoj**
https://git.sinergise.com/team-6/openeo-platform/-/issues/96

docs: https://docs.openeo.cloud/processes/?#array_append

### CODE REVIEW

File: src/pg_to_evalscript/javascript_processes/array_append.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+function array_append(arguments) {
+  const { array, value } = arguments;
```
Comment (zcernigoj) [inline src/pg_to_evalscript/javascript_processes/array_append.js:2]: `array` should be `data`

also in the test file

Comment (dgostencnik) [reply src/pg_to_evalscript/javascript_processes/array_append.js:2]: :point_up:  what he said

**Review by dgostencnik — CHANGES_REQUESTED** [review]

File: src/pg_to_evalscript/javascript_processes/array_append.js (hunk 1 above)
Comment (dgostencnik) [reply src/pg_to_evalscript/javascript_processes/array_append.js:2]: :point_up:

**Review by dgostencnik — CHANGES_REQUESTED** [review]

File: src/pg_to_evalscript/javascript_processes/array_append.js (hunk 1 above)
Comment (zcernigoj) [reply src/pg_to_evalscript/javascript_processes/array_append.js]: fixed

**Review by dgostencnik — APPROVED** [review]

**Pull request merged**